dynamic_slippage_config.json

```diff
@@ -18,7 +18,8 @@
         "FCqfQSujuPxy6V42UvafBhsysWtEq1vhjfMN1PUbgaxA",
         "8qJSyQprMC57TWKaYEmetUR3UUiTP2M3hXdcvFhkZdmv",
         "DEkqHyPN7GMRJ5cArtQFAWefqbZb33Hyf6s5iCwjEonT",
-        "2b1kV6DkPAnxd5ixfnxCpjxmKwqjjaYmCZfHsFu24GXo"
+        "2b1kV6DkPAnxd5ixfnxCpjxmKwqjjaYmCZfHsFu24GXo",
+        "USDSwr9ApdHk5bvJKMjzff41FfuX8bSxdKcR81vTwcA"
```

Comment:
I added in the other temporary list as well

https://github.com/jup-ag/dynamic-slippage-config/pull/14/commits/4f2d21a7ae8651605ad9cab4241a03f261242f2e

Reply:
acknowledged